edx_proctoring/templates/timed_exam/footer.html

```diff
@@ -2,7 +2,7 @@
 <div class="footer-sequence">
   <h4> {% trans "Can I request additional time to complete my exam?" %} </h4>
     <p>{% blocktrans %}
-      If you have disabilities or are taking the exam in difficult conditions,
+      If you have disabilities,
```

Comment:
Looking more into this, I'm not sure there are separate translation steps for this repo.

@douglashall can you confirm? Is there a separate update-the-translations release step for this repo, or does just bumping the version number in `github.txt` in edx-platform take care of things for us?